src/app/rpmostree-override-builtins.cxx

```diff
@@ -86,10 +88,29 @@ handle_override (RPMOSTreeSysroot  *sysroot_proxy,
   g_variant_dict_insert (&dict, "no-overrides", "b", opt_reset_all);
   g_variant_dict_insert (&dict, "initiating-command-line", "s", invocation->command_line);
   g_variant_dict_insert (&dict, "lock-finalization", "b", opt_lock_finalization);
+  g_variant_dict_insert (&dict, "ex-pin-from-repos", "b", opt_ex_pin_from_repos);
```

Comment:
This is entirely a client thing, so we don't need to add it to the options dict we're sending to the daemon.